google/ggrc-core
# GGRC-7032 added tip for checkbox custom attributes

# Issue description

Add hint on Acceptable options for Checkbox attributes.

# Steps to test the changes

1.Log into GGRC app(login as Admin).
2. Add a custom attribute for any object.
3. Goto Export page.
4. Select the Object Type and export the csv file.
5. Open the excel.
6. check for the added custom attribute column , here you can see the tip message
"Allowed values are:
TRUE
FALSE"

# Solution description

Updated the code function *get_custom_attr_definitions* in /src/ggrc/models/reflection.py as per requirement.

# Sanity checklist




# PR Review checklist
